config/checkstyle/suppressions.xml

```diff
@@ -6,4 +6,5 @@
 
 <suppressions>
   <suppress checks=".*" files="[\\/]build[\\/]generated[\\/].*" />
-</suppressions>
\ No newline at end of file
+  <suppress checks=".*" files="[\\/]test[\\/]java[\\/]simpleproject[\\/]SimpleRecord.java" />
```

Comment:
Why do we need this?

Reply:
`SimpleRecord` is generated code so it’s causing a handful of checkstyle issues. We need to include the generated class in the classpath in order to test if imports from classpath will work.